.circleci/config.yml

```diff
@@ -102,6 +117,9 @@ workflows:
       - chromium:
           requires:
             - ubuntu-dev
+      - discourse:
+          requires:
+            - ubuntu-dev
```

Comment:
Nit: If you agree to make the Discourse image a "Docker Hub automated build", please remove it from the CircleCI configuration.